content_scripts/cipPassword.js

```diff
@@ -348,9 +348,18 @@ var cipPassword = {
     */
     setIconPosition: function ($icon, $field)
     {
-        $icon
-            .css("top", $field.position().top + parseInt($field.css('margin-top')) + $icon.data("offset"))
-            .css("left", $field.position().left + parseInt($field.css('margin-left')) + $field.outerWidth() - $icon.data("size") - $icon.data("offset"))
+        if (window.location.hostname.match(/indiehackers.com/)) {
```

Comment:
are you really sure we need to hard code an exception here? no way to make it generic? :/